bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt

```diff
@@ -16,6 +16,7 @@
 
 package org.modelix.model.sync.bulk
 
+import mu.KotlinLogging
```

Comment:
Why was this logging library chosen?

Reply:
It's just the one we are already using across the core.